zippy/zippy.py

```diff
@@ -172,15 +172,18 @@ def workflow(self):
             else:
                 previous_stage = self.run_stage(stage, [previous_stage])
 
-    def run_zippy(self, mode='sge', mail_to=None, sge_arg_list=None):
+    def run_zippy(self, mode='sge', mail_to=None, sge_arg_list=None, pyflow_dir=None):
```

Comment:
I think we should include in the docstring a blurb that talks about how pyflow_dir overrides scratch path. This functionality seems non-obvious, and is important to understand.

Reply:
It doesn't override scratch path, though. It only overrides the output of dataDirRoot, all downstream tasks will use the regular scratch_path.